assets/services.json

```diff
@@ -13,81 +13,297 @@
+        "||awsapps.cn^",
+        "||awsapps.com.cn^",
+        "||awsautopilot.com^",
+        "||awsautoscaling.com^",
+        "||awsbraket.com^",
+        "||awscommandlineinterface.com^",
+        "||awsedstart.com^",
+        "||awseducate.com^",
+        "||awseducate.net^",
+        "||awseducate.org^",
+        "||awsglobalaccelerator.com^",
+        "||awsloft-johannesburg.com^",
+        "||awsloft-stockholm.com^",
+        "||awssecworkshops.com^",
+        "||awsstatic.cn^",
+        "||awsstatic.com^",
+        "||awsthinkbox.com^",
+        "||awstrack.me^",
+        "||awstrust.com^",
+        "||boxofficemojo.com^",
+        "||cdkworkshop.com^",
+        "||cloudfront-cn.net^",
+        "||cloudfront-test.cn^",
+        "||cloudfront.cn^",
+        "||cloudfront.net^",
+        "||containersonaws.com^",
         "||createspace.com^",
-        "||aws"
+        "||elasticbeanstalk.com^",
+        "||gameon-masters.com^",
+        "||gdansk-amazon.com^",
+        "||images-amazon.com^",
+        "||imdb.com^",
+        "||imdb.to^",
+        "||imdb^",
+        "||kindle.cn^",
+        "||kindle.co.jp^",
+        "||kindle.co.uk^",
+        "||kindle.com^",
+        "||kindle.de^",
+        "||kindle.es^",
+        "||kindle.fr^",
+        "||kindle.in^",
+        "||kindle.it^",
+        "||kindle.jp^",
+        "||kindle^",
+        "||kindleoasis.cn^",
+        "||kindleoasis.com.cn^",
+        "||kindleoasis.com^",
+        "||kindleoasis.info^",
+        "||kindleoasis.jp^",
+        "||kindleoasis.org^",
+        "||kindleoasis.us^",
+        "||kindleoasisnews.com^",
+        "||kindleproject.com^",
+        "||media-amazon.com^",
+        "||media-imdb.com^",
+        "||nwcdcloud.cn^",
+        "||nwcdcloud.com.cn^",
+        "||nwcddns.cn^",
+        "||nwcdinfosec.cn^",
+        "||prime-video.com^",
+        "||primeday.cn^",
+        "||primeday.com.cn^",
+        "||primeday.info^",
+        "||primevideo.cc^",
+        "||primevideo.com^",
+        "||primevideo.info^",
+        "||primevideo.org^",
+        "||primevideo.tv^",
+        "||route53.cn^",
+        "||sagemaker.com.cn^",
+        "||serving-sys.com^",
+        "||siege-amazon.com^",
+        "||ss2.us^",
+        "||ssl-images-amazon.com^",
+        "||thinkboxsoftware.com^",
+        "||ueberamazon.de^",
+        "||xn--cckwcxetd^",
+        "||xn--jlq480n2rg^",
+        "||yamaxun.cn^",
+        "||yamaxun.com^",
+        "||yamaxun^",
+        "||z.cn^",
+        "||zappos^",
+        "/.+\\.awsdns-[0-9][0-9]\\.co\\.uk$/",
+        "/.+\\.awsdns-[0-9][0-9]\\.com$/",
+        "/.+\\.awsdns-[0-9][0-9]\\.net$/",
+        "/.+\\.awsdns-[0-9][0-9]\\.org$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.biz$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.com$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.net$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.top$/",
+        "/.+\\.awsdns-cn-[0-9][a-e0-9]\\.cn$/",
+        "/^images-cn-8\\.ssl-images-amazon\\.com$/",
```

Comment:
Replace with a regular rule: `||images-cn-8.ssl-images-amazon.com^`

Please avoid regex rules when possible.

Reply:
Ok, but I want to ask, why avoid using regex?